src/components/Contacts/ContactDetails/ContactDetails.test.tsx

```diff
@@ -11,19 +13,21 @@ const onClose = jest.fn();
 
 describe('ContactDetails', () => {
   it('should show loading state', async () => {
-    const { getByTestId } = render(
+    const { queryByTestId } = render(
```

Comment:
I think I misled you. 😞  Sorry... I meant `getBy` but I said use `findBy` instead of `queryBy` https://testing-library.com/docs/queries/about#:~:text=Summary%20Table has:

Type of Query | 0 Matches | 1 Match | >1 Matches | Retry (Async/Await)
-- | -- | -- | -- | --
Single Element |   |   |   |  
getBy... | Throw error | Return element | Throw error | No
queryBy... | Return null | Return element | Throw error | No
findBy... | Throw error | Return element | Throw error | Yes
Multiple Elements |   |   |   |  
getAllBy... | Throw | Return array | Return array | No
queryAllBy... | Return [] | Return array | Return array | No
findAllBy... | Throw | Return array | Return array | Yes

I think the takeaway from that is use `queryBy` when it might not be on the page, use `findBy` when it might not be on the page immediately, and `getBy` otherwise. I think we should use `getBy` by default unless more features are needed.

They all work, so it's not a big deal as long as you get the test working but it's probably best to use them as they were designed.

Reply:
Gotcha, thank you for that clarification!  I can go through and use `getBy` instead, I'd rather stay consistent to our standards.  I'd also rather it not be asynchronous unless it needs to be.